x/twasm/keeper/handler_plugin.go

```diff
@@ -49,22 +58,25 @@ type TgradeHandler struct {
 	minter                 minter
 	govRouter              govtypes.Router
 	consensusParamsUpdater ConsensusParamsUpdater
+	staker                 staker
 }
 
 // NewTgradeHandler constructor
 func NewTgradeHandler(
 	cdc codec.Codec,
 	keeper TgradeWasmHandlerKeeper,
-	bankKeeper minter,
+	minter minter,
```

Comment:
can you merge staker + minter into bankKeeper? I think they are both implemented in the sdk bank.keeper

Reply:
They are. Just segregated them for tests. Will try to join them.